recipe/install-py-xgboost.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+. activate "${PREFIX}"
```

Comment:
You pribably need `activate_in_script` as we are using it in the `python-feedstock` instead: https://github.com/conda-forge/python-feedstock/blob/8fae6d37ed0e54039822db89a78ab38a76db0110/recipe/meta.yaml#L129

```suggestion
```

Reply:
Would this replace the need for `install-libxgboost.bat` as it was suggested in the comment above?

Reply:
Using the generic wrapper suggested by @bollwyvl should work then, yes.